client/app/config/steps/password.coffee

```diff
@@ -1,5 +1,16 @@
+jQuery = require 'jquery'
+
 module.exports = {
     name: 'password',
     route: 'password',
     view : 'steps/password'
+
+    # If OK, return null
+    # if not return an Array of errors
+    validate: (data) ->
+        return null
+
+    submit: (data) ->
```

Comment:
I think we should not allow to override `submit` and create another method `save` called by `submit` like it already does here with `validate`. It is a more hookable approach and it's actually what I've done here : https://github.com/cozy/cozy-proxy/pull/324/files#diff-7f3ff7953dafb8bab4ffe2534bbae502R5

It also simplifies the code in onboarding.initialize, as we are not keeping native submit in a closure.

As it is done in another PR, modification must be done in this one or #324. It depends on which one will be merged first.